cleanup.go

```diff
@@ -8,29 +8,76 @@ import (
 	"time"
 )
 
-// terminateOptions is a type that holds the options for terminating a container.
-type terminateOptions struct {
-	ctx     context.Context
-	timeout *time.Duration
-	volumes []string
+// DefaultTimeout for termination
+var DefaultTimeout = 10 * time.Second
```

Comment:
bug: sorry mentioned this previously but missed this in the last review, with it being exposed it can introduce race condition.
```suggestion
```